### PR TITLE
Add Force Login to the list

### PR DESCRIPTION
**Pull request by jfussion**
## Effect
PR includes the following changes:
- Added Force Login WordPress plugin to the [modules-plugins-known-issues](https://pantheon.io/docs/modules-plugins-known-issues) page

### CODE REVIEW

File: source/_docs/modules-plugins-known-issues.md [hunk 1]
```diff
@@ -339,6 +339,12 @@ For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_nam
 **Solution**: See [Caching: Advanced Topics](/docs/caching-advanced-topics/) for details on how to bypass the platform page cache.
 <hr>
 
+### [Force Login](https://wordpress.org/plugins/wp-force-login/){.external}
+**Issue**: This plugin appends a port number using `$_SERVER['SERVER_PORT']` at the end of the URL when the user logs in to the site.
```
Comment (alexfornuto) [inline source/_docs/modules-plugins-known-issues.md]: Is it always the same port number? What number is it? What error does this cause, if any?

File: source/_docs/modules-plugins-known-issues.md [hunk 2]
```diff
@@ -339,6 +339,12 @@ For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_nam
...
+
+**Solution**: See [Set SERVER_PORT Corectly](https://pantheon.io/docs/server_name-and-server_port/#set-server_port-correctly).
```
Comment (alexfornuto) [inline source/_docs/modules-plugins-known-issues.md]: Would the customer need to replace any values in the example snippet linked? Which values, and replaced to what?

File: source/_docs/modules-plugins-known-issues.md (hunk 1 above)
Comment (jfussion) [reply source/_docs/modules-plugins-known-issues.md]: It depends on what the port number the appserver is using. You can check that port number on the debug pannel of the site
children s hospital of philadelphia pantheon dashboard 2018-07-28 09-48-37

File: source/_docs/modules-plugins-known-issues.md (hunk 2 above)
Comment (jfussion) [reply source/_docs/modules-plugins-known-issues.md]: No need to modify the snippet.

**Comment by jfussion** [comment]
Here's the line from Force login plugin code where it uses the said environment variable:
`wp-force-login/wp-force-login.php:31:      $url .= in_array( $_SERVER['SERVER_PORT'], array('80', '443') ) ? '' : ':' . $_SERVER['SERVER_PORT'];`

File: source/_docs/modules-plugins-known-issues.md (hunk 1 above)
Comment (jfussion) [reply source/_docs/modules-plugins-known-issues.md]: @alexfornuto It's not the same port. The port number is random based on the port number used by the appserver as shown in the image. The result is a white page.

You can test it here: http://force-login-jts-wp.pantheonsite.io/hello-world
user: josephpantheon
pass: WbjgZ(Bt#hVKq3zDUT)uekR2

**Pull request merged**